flask_restless/views.py

```diff
@@ -1058,6 +1061,31 @@ def _search(self):
         for preprocessor in self.preprocessors['GET_MANY']:
             preprocessor(search_params=search_params)
 
+        # resolve date-strings as required by the model
+        for param in search_params.get("filters", list()):
+            if all(key in param for key in ('name', 'val')):
```

Comment:
Two conditions is not enough to warrant an `all` invocation. I prefer

    if 'name' in param and 'val' in param:

for its simplicity and readability.